lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/DBOVerificationDAOImpl.java

```diff
@@ -56,8 +58,35 @@
 	@Autowired
 	private IdGenerator idGenerator;
 
+	// get the latest submission for a user
+	// select * from verification_submission v where 
+	// v.created_by=? and 
+	// v.created_on=
+	// (select max(v2.created_on) from verification_submission v2 where v2.created_by=v.created_by)
+	private static final String LATEST_VERIFICATION_SUBMISSION_SQL = 
```

Comment:
add created_on to the primary key, i.e. PK = <verification_id, created_on>  This ensures that this query may only return one row.